app/selenium_ui/jsd/modules_agents.py

```diff
@@ -66,3 +67,13 @@ def measure():
         browse_projects_page.go_to()
         browse_projects_page.wait_for_page_loaded()
     measure()
+
+
+def browse_project_customers_page(webdriver, datasets):
+    browse_customers = BrowseCustomers(webdriver)
+
+    @print_timing('selenium_browse_project_customers_page')
+    def measure():
+        browse_customers.go_to_url(UrlManager(project_key=datasets['project_key']).browse_project_customers_page())
```

Comment:
Please create an appropriate method in the Page class. So it should be like `CustomersPage` with argument as project_key, `customers_page = Customers(project_key=...)` and `customers_page.go_to`

Reply:
Updated